stock_orderpoint_generator/models/orderpoint_template.py

```diff
@@ -80,6 +80,22 @@ class OrderpointTemplate(models.Model):
         "scheduled action for every product in this list.",
     )
     auto_last_generation = fields.Datetime(string="Last Automatic Generation")
+    trigger = fields.Selection(
+        selection=[
+            ("auto", "Auto"),
+            ("manual", "Manual"),
```

Comment:
@aayartsev Could you explain the aim of this ? As manual triggering for orderpoint creation... is manual creation

Reply:
@rousseldenis Value of this field will be copy to record wich will be created from this template.

Reply:
@aayartsev Could you add your changes in the USAGE.rst ? Thanks

Reply:
Hi @rousseldenis  )
I have added our changes to USAGE.rst
please check)